code/modules/mob/living/carbon/human/life.dm

```diff
@@ -311,8 +311,18 @@
 	// Tissues die without blood circulation
 	adjustBruteLoss(2)
 
-
-
+/mob/living/carbon/human/proc/handle_nearby_food()
+	if(!client) return
+	if(world.time - last_eaten < 30 SECONDS) return //30 seconds after eating food you start getting hungry from seeing food again
+	if(world.time - last_seen_nearby_food < 15 SECONDS) return
+	if(nutrition >= NUTRITION_LEVEL_WELL_FED) return
+
+	for(var/obj/item/reagent_containers/food/F in view(client.view, get_turf(src)))
```

Comment:
i'm not qualified (see: maintainer) to be complaining about this stuff but is calling view on a 15x15 area just to find food every second for every connected human player really worth a meme feature?

Reply:
no but we can leave this PR open for a while to indulge it

Reply:
food is costly too, have you seen grocery prices these days (outside the cat food aisle)???